app/src/pages/inside/manualLaunchesPage/manualLaunchDetailsPage/manualLaunchDetailsPage.scss

```
@@ -23,49 +25,18 @@
-    &.active {
-      svg {
-        color: var(--rp-ui-base-topaz);
-      }
-
-      &:hover {
-        background-color: var(--rp-ui-base-bg-200);
-      }
-    }
+  &__filter-icon {
+    @include mixins.header-toolbar-filter-button;
+    flex-shrink: 0;
```

Stylelint (17.6.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)